### PR TITLE
Bug fix/make ttl index work with datestrings too

### PR DESCRIPTION
**Pull request by jsteemann**
Make TTL index work with date strings too

### CODE REVIEW

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/3109/

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/3121/

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/3127/

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/3134/

File: Documentation/Books/Manual/Indexing/IndexBasics.md [hunk 1]
```diff
@@ -283,17 +285,24 @@ out, so queries may still find and return documents that have already expired. T
...
-there is `Date.now()`, to calculate it from an arbitrary date string, there is 
-`Date.getTime()`.
-
-In case the index attribute does not contain a numeric value, the document will not
-be stored in the TTL index and thus will not become a candidate for expiration and 
-removal. Providing either a non-numeric value or even no value for the index attribute
-is a supported way of keeping documents from being expired and removed.
+specified in milliseconds since January 1st 1970 (Unix timestamp). To calculate the current 
+timestamp from JavaScript in this format, there is `Date.now() / 1000`, to calculate it 
+from an arbitrary date string, there is `Date.getTime() / 1000`.
```
Comment (graetzer) [inline Documentation/Books/Manual/Indexing/IndexBasics.md]:
```suggestion
from an arbitrary Date instance, there is `Date.getTime() / 1000`.
```

Comment (graetzer) [reply Documentation/Books/Manual/Indexing/IndexBasics.md]:
```
var moonLanding = new Date('July 20, 69 00:20:18 GMT+00:00');
moonLanding.getTime()
```

File: Documentation/Books/Manual/Indexing/Ttl.md [hunk 2]
```diff
@@ -31,19 +33,28 @@ out, so queries may still find and return documents that have already expired. T
...
-
+  
 Please note that the numeric date time values for the index attribute should be 
-specified in milliseconds since January 1st 1970 (Unix timestamp with millisecond
-precision). To calculate the current timestamp from JavaScript in this format, 
-there is `Date.now()`, to calculate it from an arbitrary date string, there is 
-`Date.getTime()`.
+specified in milliseconds since January 1st 1970 (Unix timestamp). To calculate the current 
+timestamp from JavaScript in this format, there is `Date.now() / 1000`, to calculate it 
+from an arbitrary date string, there is `Date.getTime() / 1000`.
```
Comment (graetzer) [inline Documentation/Books/Manual/Indexing/Ttl.md]: same as before

**Pull request merged**